src/IC-211/kotlin/io/kotest/plugin/intellij/files.kt

```diff
@@ -0,0 +1,13 @@
+package io.kotest.plugin.intellij
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.psi.search.FilenameIndex
+import com.intellij.psi.search.GlobalSearchScope
+import io.kotest.plugin.intellij.toolwindow.TagsFilename
+
+fun findFiles(project: Project): List<VirtualFile> {
+   return FilenameIndex
+      .getVirtualFilesByName(project, TagsFilename, false, GlobalSearchScope.allScope(project))
+      .toList()
+}
```

Comment:
Identical on all platforms, can't it be put in `main` instead?

Reply:
Good call.